SECURITY.md

```diff
@@ -0,0 +1,16 @@
+# Security Policy
+
+Security is very important for DocArray and its community. 
+
+## Supported Versions
+
+The latest versions of DocArray are supported.
```

Comment:
Is this specific enough? Do we mean the latest version, the latest two versions, or the latest five versions? Minor versions or patch versions?

Reply:
I copy past from jina core tbh.

I think that it means that the last minor version is supported as our plan now is not maintain older minor version (ex 3.0) since we merge bug fix in the newer version